examples/next-login/src/pages/home/signin.tsx

```diff
@@ -0,0 +1,207 @@
+import { Text, Stack, Alert } from '@ultraviolet/ui'
+import { Icon } from '@ultraviolet/icons'
+import {
+  Form,
+  TextInputFieldV2,
+  TextInputField,
+  Submit,
+  useForm,
+  RadioField,
+  DateField,
+  FormErrors,
+} from '@ultraviolet/form'
+import { Theme, css, useTheme } from '@emotion/react'
+import { useState } from 'react'
+
+type FormValues = {
+  email: string
+  password: string
+  gender: string
+  firstname: string
+  lastname: string
+  birthdate: Date
+}
+
+const emailRegex = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
+const mockErrors: FormErrors = {
+  maxDate: ({ maxDate }) => `Date must be lower than ${maxDate?.toString()}`,
+  maxLength: ({ maxLength }) =>
+    `This field should have a length lower than ${maxLength}`,
+  minDate: ({ minDate }) => `Date must be greater than ${minDate?.toString()}`,
+  minLength: ({ minLength }) =>
+    `This field should have a length greater than ${minLength}`,
+  pattern: () => `This field should match the regex`,
+  required: () => 'This field is required',
+  max: ({ max }) => `This field is too high (maximum is : ${max})`,
+  min: ({ min }) => `This field is too low (minimum is: ${min})`,
+}
+
+const bodyStyle = (theme: Theme) => css`
```

Comment:
Sam here you could do styled out of those css and remove `className` in the code